geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaDataStoreParams.scala

```diff
@@ -113,6 +113,16 @@ object KafkaDataStoreParams extends NamespaceParams {
       readWrite = ReadWriteFlag.ReadOnly
     )
 
+
+  // use of Long isn't working, will capture as Integer and convert to long
```

Comment:
did you try `java.lang.Long` instead of the primitive `Long`?

Reply:
working through your edits today - I did not try java.lang.Long (not being in the scala space a lot, I don't always remember to check these nuances, thanks.

Reply:
config with java.lang.Long does work (and still use map to convert to scala Long) - thanks.